ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+
+Thank you for being interested inv nvm, here is a quick form for problem reporting, please help us fill it if you are going to open an issue with bug/problem report, if you are just going to suggest feature or something else but not like this, please just remove it, thanks!
+
+- Operating system and version:
+
+- `nvm debug` output:
+```
+```
+
+- How did you install nvm (e.g. homebrew, install script in readme):
```

Comment:
let's make sure homebrew is last in the list. also, there should be a question mark after "nvm" here